Wyam.Core/Modules/CopyFiles.cs

```diff
@@ -106,7 +106,7 @@ public IEnumerable<IDocument> Execute(IReadOnlyList<IDocument> inputs, IExecutio
                             .Select(file =>
                             {
                                 string destination = _destinationPath == null
-                                    ? Path.Combine(context.OutputFolder, PathHelper.GetRelativePath(Path.GetDirectoryName(path), Path.GetDirectoryName(file)), Path.GetFileName(file)) 
+                                    ? Path.Combine(context.OutputFolder, PathHelper.GetRelativePath(context.InputFolder, Path.GetDirectoryName(file)), Path.GetFileName(file)) 
```

Comment:
I think checking for relative path is fine. Allowing ascention up the directory tree seems like such an edge case, I don't think we need to support it - at least not right now.